{
  "uuid": "9f046b4d-3dce-47b1-88e0-85b247e95802",
  "name": "OdinUiTests.Tests.Activity.TaskWithSolutionStudentTests.Наличие ошибки при удалении решения",
  "children": [
    "fa940425-5be3-4f59-aac0-a0fc6a0de707"
  ],
  "befores": [],
  "afters": [],
  "links": [],
  "start": 1704516444574,
  "stop": 1704516448325
}